README.md

```diff
@@ -23,6 +23,20 @@ See <https://community.flake.parts/services-flake/start>
 
 List of supported services is available at https://community.flake.parts/services-flake/services
 
+## Comparison with similar approaches
+
+| | services-flake | [devenv](https://devenv.sh/) |
+| --- | --- | --- |
+| macOS support | ✔️  | ✔️  |
+| Pure Flakes | ✔️  | ❌[^1]  |
+| Share services across flakes | ✔️[^2]  | ❌  |
```

Comment:
This maybe inaccurate.

cf. https://devenv.sh/guides/using-with-flake-parts/#import-a-devenv-module

But I haven't used devenv enough to say if its module system works without any idiosyncratic shortcomings.

Reply:
Based on what I see, if you are able to `import` a module, you should be able to share services even using `devenv`.

Reply:
And I tried it out:

Here’s the `perSystem` block from my flake:

<img width="460" alt="Screenshot 2024-03-08 at 11 43 16 AM" src="https://github.com/juspay/services-flake/assets/23645788/a34e5678-2f98-4f4c-beb6-f429528a58f4">


And this is the `devenv-module.nix`:

<img width="432" alt="Screenshot 2024-03-08 at 11 43 43 AM" src="https://github.com/juspay/services-flake/assets/23645788/7ae34a69-eb0e-4e9a-8066-2e56e91fe660">


I do use `nix develop —impure` followed by `devenv up` to run them but in terms of sharing services, it works, i.e if you are willing to stick to devenv cli.

Reply:
>  we can't do the following, right?

Yes, hope I haven’t missed something. Let me give it one more shot to be certain, don’t want to make false claims because of missing some details.

Reply:
@shivaraj-bh https://discord.com/channels/1036369714731036712/1036369758024650792/1215626814710620190

Reply:
I haven't gotten any clear and unambiguous from the devenv community, so let's just merge this. If they correct us, we can change the README obviously.